parachain/pallets/control/src/lib.rs

```diff
@@ -76,14 +78,37 @@ pub mod pallet {
 		Upgrade { impl_address: H160, impl_code_hash: H256, params_hash: Option<H256> },
 		/// An CreateAgent message was sent to the Gateway
 		CreateAgent { location: Box<MultiLocation>, agent_id: AgentId },
+		/// An CreateChannel message was sent to the Gateway
+		CreateChannel { location: Box<MultiLocation>, para_id: ParaId, agent_id: AgentId },
+		/// An UpdateChannel message was sent to the Gateway
+		UpdateChannel {
+			location: Box<MultiLocation>,
+			para_id: ParaId,
+			agent_id: AgentId,
+			mode: OperatingMode,
+			fee: u128,
+			reward: u128,
+		},
+		/// An SetOperatingMode message was sent to the Gateway
+		SetOperatingMode { mode: OperatingMode },
+		/// An TransferNativeFromAgent message was sent to the Gateway
+		TransferNativeFromAgent {
+			location: Box<MultiLocation>,
```

Comment:
Ditto, not required to log the origin location. That is only required for CreateAgent.